tests/letstest/README.md

```diff
@@ -28,8 +28,7 @@ sudo apt install awscli python-yaml python-boto3 fabric
 [interactive: enter secrets for IAM role]
 >aws ec2 create-key-pair --profile <profile name> --key-name <key name> --query 'KeyMaterial' --output text > whatever/path/you/want.pem
 ```
-Note: you can pick any file name for `<key-name>`, but it must have a `.pem` extension. Also, whatever you pick for `<profle name>` will be shown
-to other users with AWS access.
+Note: whatever you pick for `<key name>` will be shown to other users with AWS access.
```

Comment:
Nice catch! I thought the profile was name was shown but you're right that it's the key name. With this in mind, I have one more suggested change...